### DISCUSSION TITLE
Spotbugs does not report bugs

### DISCUSSION THREAD

**Discussion by mariometushev** [spotbugs/spotbugs#2207] [Q&A]
Hi guys, I have a question.

I'm using "com.github.spotbugs" plugin for my project, his version is 5.0.7

This is my Gradle configuration:
![spotbugs](https://user-images.githubusercontent.com/96615848/194574329-298d3c5f-2fbb-4aea-bbaf-00333b874b77.PNG)

When I see the files generated from spotbugs, for every .class file, reported bugs are 0 and for the whole project they remain 0. 
The expected bugs count produced when I'm using Ant not Gradle is 20,000. 

![nowarns](https://user-images.githubusercontent.com/96615848/194576373-8886d01c-c967-4d73-88d9-38051a255ff0.PNG)

Can someone explain what I'm doing wrong with the configuration :)

**Comment by ThrawnCA** [comment]
Does buildDir need braces?

**Reply by mariometushev** [reply]
I don't think that's the problem. Spotbugs creates a report directory but cannot find the bugs
